libs/index.md

```diff
@@ -198,6 +198,9 @@ Available query params:
       ['yearly income', '/opencollective/yearly/webpack'],
     ]
   }
+  window.docPages = {
+    packagephobia: 'packagephobia',
```

Comment:
I guess this could just be an array?

Reply:
Random thought: could the future also include some sort of parameterized popin that generates complete markdown (badge + link)? 😃

Reply:
Yep, a badge builder 💪

When I pulled in `lit-html`(thought about jsx before), it's on the roadmap  😄